### PR TITLE
feat(GUI): center content independent to window resolution

### PR DESCRIPTION
**Pull request by Shou**
We pave way for different sizes of Etcher windows by dynamically
centering the content with flexbox.

Connects-To: #2263
Closes: #2241
Change-Type: patch
Changelog-Entry: Increase the vertical window size to 480px.

### CODE REVIEW

**Comment by Shou** [comment]
image
vs
image

**Comment by jhermsmeier** [comment]
@konmouz @Shou @jviotti do we want to hide this behind a Etcher Pro feature flag? I think it looks a bit empty on desktop

**Comment by lurch** [comment]
Should this also change the version-number sent to the online success-banner URL, so that a (different) success banner can be displayed that takes full advantage if the increased space? (while the old smaller-sized success banner still gets displayed by older versions of Etcher)

**Comment by Shou** [comment]
@jhermsmeier the extra space should be used by https://github.com/resin-io/etcher/issues/2263 once that's implemented so maybe this shouldn't be merged until there's progress on that so we can still do releases if we have to.

**Comment by Shou** [comment]
@lurch I wonder if we should increase the API variable given that the interface hasn't changed? Instead we could check the version string in the banner? Or maybe that's a slippery slope into a tangled mess.

**Comment by lurch** [comment]
Probably easier to just check a single version-variable than it is to do `if ((programVersion == xxx) || (programVersion == yyy) || (programVersion == zzz) || ...` ?

**Comment by jhermsmeier** [comment]
> so maybe this shouldn't be merged until there's progress on that so we can still do releases if we have to

@Shou can you add a "Depends on" to the PR description, so we're immediately aware of it (without reading the whole thread) when looking at this?

**Comment by Shou** [comment]
I'll go with the API version increase. Also here's what it looks like right now with the current banner.

image

Gonna push these changes soon, and more screenshots to follow.

**Comment by jviotti** [comment]
@jhermsmeier @Shou We should not be hiding this behind any Etcher Pro specific environment variable. If the new window feels empty, then we can revise the design.

**Comment by jhermsmeier** [comment]
@jviotti yeah, but no need to revise the design with what @Shou mentioned, as https://github.com/resin-io/etcher/issues/2263 will fill the space – I think we should just wait until that's done before we merge this, as @Shou suggested, to avoid awkward looks

**Comment by Shou** [comment]
Yup, I probably have to make some small adjustments after rebasing as well, but it's mostly done.

File: lib/gui/app/components/safe-webview.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const API_VERSION_PARAM = 'api-version'
  * This version number is exposed to the banner such that it can determine what
  * features are safe to utilize.
  */
-const API_VERSION = 1
+const API_VERSION = 2
```
Comment (lurch) [inline lib/gui/app/components/safe-webview.js]: Perhaps it'd be worth adding some comments, clarifying what changes each of the different API_VERSIONs signify?

Comment (Shou) [reply lib/gui/app/components/safe-webview.js]: That's a good idea, but it'd grow with every version change, instead maybe we could just have a comment like below.

```
// For history use: git blame -L n
```

Comment (lurch) [reply lib/gui/app/components/safe-webview.js]: > but it'd grow with every version change

I guess it depends on how often you expect `API_VERSION` to change...

**Pull request merged**